README.md

```diff
@@ -40,6 +40,7 @@ please open a pull request adding it below in alphabetical order with a URL and
 description of your deviance.
 
 * [Lumos Labs](https://github.com/lumoslabs/sqlstyle.guide) - removed the river, Redshift, etc. [www](http://engineering.lumosity.com/sqlstyle.guide)
+* [nkurigit](https://github.com/nkurigit/sqlstyle.guide) - for Japanese translation
```

Comment:
I was thinking that this would be more of a contributors list rather than a link to your fork - if the translation changes then it will be changed as a pull request against the `treffynnon/sqlstyle.guide` repo rather than there.

I was thinking something more along the lines of:

```markdown
## Translations

* [Japanese/日本語](http://www.sqlstyle.guide/jp) - [Nkuri](https://github.com/nkurigit)
```